src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageItemViewModel.cs

```diff
@@ -145,7 +145,7 @@ public NuGetVersion LatestVersion
                     if (_latestVersion != null)
                     {
                         var displayVersion = new DisplayVersion(_latestVersion, string.Empty);
-                        string toolTipText = PackageLevel == PackageLevel.Transitive ? Resources.ToolTip_TransitiveDependency : Resources.ToolTip_LatestVersion;
+                        string toolTipText = PackageLevel == PackageLevel.Transitive ? Resources.ToolTip_TransitiveDependencyVersion : Resources.ToolTip_LatestVersion;
```

Comment:
It seems ToolTip_LatestVersion has one {0} argument. Is this argument used lated?

Reply:
Yes. See: https://github.com/NuGet/NuGet.Client/blob/0f4466a936e8c0daa5a6efae1800f3f9c41b4a8e/src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageItemViewModel.cs#L151

Reply:
In this line it sets a variable called toolTipText to either the ToolTip_TransitiveDependencyVersion  one or the ToolTip_LatestVersion one. Both have the {0} parameter. Then toolTipText is used in the string.Format below.